assets/xml/objects/gameplay_keep.xml

```diff
@@ -935,7 +935,7 @@
         <DList Name="gameplay_keep_DL_029BE8" Offset="0x29BE8" />
         <DList Name="gameplay_keep_DL_029CB0" Offset="0x29CB0" />
         <DList Name="gameplay_keep_DL_029CF0" Offset="0x29CF0" />
-        <DList Name="gameplay_keep_DL_029D10" Offset="0x29D10" />
+        <DList Name="gMushroomGlow" Offset="0x29D10" /> <!-- Original name is "gameplay_keep_DL_029D10" -->
```

Comment:
I still don't think this solves the problem (especially since it can get mixed up with animations where we know the original name.

Reply:
I will just return it to the original name and leave a comment that it creates a purple glow in `obj_kinoko`